README.md

```diff
@@ -50,13 +50,13 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 # Usage
 At the heart of Brigadier, you need a `CommandDispatcher<S>`, where `<S>` is any custom object you choose to identify a "command source".
 
-A command dispatcher holds a "command tree", which are a series of `CommandNode` which represent the various possible syntax that form a valid command.
+A command dispatcher holds a "command tree", which are a series of `CommandNode` which represents the various possible syntax that forms a valid command.
```

Comment:
I'm not sure I agree with this.  The issue here is that `CommandNode` isn't written as plural, but it would be for "series of"; you say "a series of objects which represent something" and not "a series of object which represents something".

Reply:
Ok, I'm not an expert in this, as I'm not an English teacher but a programmer, and English is actually my 2nd language, Chinese being my 1st language, so take this with a grain of salt.

But looking at the sentence, I think maybe turning

> A command dispatcher holds a "command tree", which are a series of `CommandNode` **which represent** the various possible syntax that form a valid command.

to

> A command dispatcher holds a "command tree", which are a series of `CommandNode` **representing** the various possible syntax that form a valid command.

might work better in this case.

Reply:
Ok, I spent some time with others thinking about this, and we ended up with this. 

> A command dispatcher holds a "command tree", which is a series of `CommandNode`s that represent the various possible syntax options that form a valid command.

Is this Good?

Reply:
uh, you should have consulted an SAT writing (grammar) teacher before fixing grammar then.